TwilioVerifyDemo/TwilioVerifyDemo/ChallengeDetail/Presenter/ChallengeDetailPresenter.swift

```diff
@@ -45,6 +46,13 @@ class ChallengeDetailPresenter {
     self.twilioVerify = twilioVerify
     self.challengeSid = challengeSid
     self.factorSid = factorSid
+    
+    // TODO: Call cache to check for pending challenges. (TwilioVerifyDemoCache)
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/64d430897a2df4dbbabee1ff15ea40919450eaae/TwilioVerifyDemo/TwilioVerifyDemo/ChallengeDetail/Presenter/ChallengeDetailPresenter.swift#L50"></span>TODOs should be resolved (Call cache to check for pendin…).<br />
<code>todo</code> <code>ChallengeDetailPresenter.swift:50</code></td>
    </tr>
  </tbody>
</table>